css/style.css

```diff
@@ -163,7 +163,7 @@ Main components
     }
 
     .cd-tabs-navigation a.selected:not(#reload) {
-        box-shadow: inset 0 5px 0 #31ac63; /* Selected tab marker colour */
+        box-shadow: inset 0 5px 0 CornflowerBlue; /* Selected tab marker colour */

```

Comment:
#6495ed is the hex - don't want to risk some browser not implementing the shorthand.